spec/factories/content_items_factory.rb

```diff
@@ -2,6 +2,6 @@
   factory :content_item do
     sequence(:content_id) { |index| "7776ddf3-f918-5f32-bf18-dc1ced2eeeb#{index}" }
     link "api/content/item/path"
-    public_updated_at Time.parse("2016-11-01 11:20:45.481868")
+    public_updated_at { Time.at(0 + rand * (Time.now.to_f - 0.to_f)) }
```

Comment:
I thought about this, but the range diff was small and somewhat confusing.. rather than do this I prefer being a bit more explicit.